src/com/twu/biblioteca/MainMenu.java

```diff
@@ -1,40 +1,62 @@
 package com.twu.biblioteca;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class MainMenu {
 
-    public void welcome() {
-        System.out.println("Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!");
+    private final String welcome = "Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!";
+    private final String invalidMessage = "Please select a valid option!";
+    private final String goodBye = "Goodbye!";
+
+   ArrayList<String> books = new ArrayList<String>(Arrays.asList(
+           "1. Clean Code | Robert C. Martin | 2008",
+           "2. Don't Make Me Think | Steve Krug | 2000",
+           "3. Test Driven Development | Kent Beck | 2000"));
+
+   ArrayList<String> options = new ArrayList<String>(Arrays.asList(
+           "Please select an option from the following:",
```

Comment:
I'm getting _really_ picky here, but is this first line an option?  It's in an array called `options` but is it the same type of thing as the other elements, or just used at the same time?

Reply:
Ah no, it's not an option, it's a message to provide context to the user. It should be outside of the array. I will move it out.